tensorflow/core/kernels/rnn/lstm_ops.cc

```diff
@@ -1046,7 +1046,7 @@ class BlockLSTMOp : public OpKernel {
       Tensor co_tensor = slicer.OutputSlice(co_out, t, "co_out");
       Tensor h_tensor = slicer.OutputSlice(h_out, t, "h_out");
 
-      functor::LSTMBlockCellFprop<Device, T, USE_CUBLAS, gate_layout>(
+      OP_REQUIRES_OK(ctx, functor::LSTMBlockCellFprop<Device, T, USE_CUBLAS, gate_layout>(
```

Comment:
Since this is in a helper, you must also validate `ctx->Status` after each call of the helper, since `OP_REQUIRES_OK` in the end is just setting `ctx->Status` and then `return`-ing

Reply:
So I actually altered this in a more recent commit because the build fails when I call `OP_REQUIRES_OK` here with an `error: invalid use of void expression`. Instead, I implemented a check on `ctx->Status` like you're saying.